packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,63 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+function range(file, node, source, color, CONTEXT = 2) {
+  if (file === null || node === null || source === null || color == null) {
```

Comment:
This checks for explicitly passing `null`. If you want to bail on either explicitly passing `null` or just omitting the arguments, you should use `== null`.